src/__test__/linked-list-test.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
```

Comment:
This file should be named linked-list.test.js - Jest only looks for files with the .test.js extension after it. That's why your tests aren't running.